src/applications/travel-pay/components/TravelClaimDetails.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from 'react';
+import { useParams, useLocation } from 'react-router-dom';
+import { scrollToTop } from 'platform/utilities/ui/scroll';
+import { focusElement } from 'platform/utilities/ui';
+
+import { apiRequest } from '@department-of-veterans-affairs/platform-utilities/api';
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+
+import { formatDateTime } from '../util/dates';
+import BreadCrumbs from './Breadcrumbs';
+
+export default function TravelClaimDetails() {
+  const { id } = useParams();
+  const location = useLocation();
+  const { claimDetailsProps = null } = location.state ?? {};
+  const [claimDetails, setClaimDetails] = useState(claimDetailsProps);
+  const [isLoading, setIsLoading] = useState(!claimDetailsProps);
+  const [claimsError, setClaimsError] = useState(null);
+  const BTSSS_PORTAL_URL = 'https://dvagov-btsss.dynamics365portals.us/';
+
+  console.log(claimDetails); // eslint-disable-line no-console
```

Comment:
Why is the linter disabled here? Let's remove this to avoid console output where not necessary.

Reply:
This is just a code snippet I use in local dev that needs to be cleaned up